pandas/io/common.py

```diff
@@ -452,6 +480,12 @@ def get_handle(
         need_text_wrapping = (BufferedIOBase, RawIOBase, S3File)
     except ImportError:
         need_text_wrapping = (BufferedIOBase, RawIOBase)
+    try:
+        from fsspec.spec import AbstractFileSystem
+
+        need_text_wrapping = tuple(list(need_text_wrapping) + [AbstractFileSystem])
+    except ImportError:
+        pass
```

Comment:
is there a good way in python/pytest to make `fsspec` inaccessible for a specific test to make sure that both branches are covered?